transfer-data/introduction.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Introduction'
+description: 'This section explains how to set up a flow to transfer and transform data from an Axiom dataset to a destination.'
+---
+
+<Note>
+Flow is currently in preview. To try it out, [sign up for a free preview](https://forms.axiom.co/flow).
+</Note>
+
+A flow is a way to transfer and transform data from an Axiom dataset to a destination that you choose. For example, you can use flows to retain the complete record of all events in Axiom while transferring high-value app events to a costly SIEM.
+
+The **Flows** tab allows you to move and transform data using APL and provides you with a unified experience that streamlines your data workflows.
+
+## Elements of a flow
+
+A flow consists of three elements:
+- A source. This is the Axiom dataset from which you transfer and transform data.
+- A transformation. This is the APL query you use to shape your data.
+- A destination. This is where you transfer the data.
+
+GIF
+
+After creating a flow, replay it for a specified time range to transfer the matching and transformed data to the destination.
```

Comment:
This might be a better space to introduce concepts of one-time, scheduled, and continuous flows, with link back to that context in Flows page if preferred.

Reply:
@dominicchapman Thanks. What do you think?

```
After creating a flow, replay it for a specified time range to transfer the matching and transformed data to the destination. There are three types of flows:
- One-time flows transfer data to the destination once when you manually initiate the replay.
- Scheduled flows transfer data to the destination once at a scheduled time.
- Continuous flows transfer data to the destination on a rolling basis, when Axiom receives new events that match your APL query.

The preview only supports one-time flows.
```

Reply:
We talked about some changes in our meeting. 
If meeting recording is helpful, nudge me and I’ll link it on Slack.

Reply:
@dominicchapman Thank you! I've implemented your suggestions here: https://github.com/axiomhq/docs/pull/31/commits/7f5f900867a791b499b7bea6ca58303a28208e90
Please let me know if this has become better or if I should add more information about types of flows.

Reply:
I revised the video with one from Monitorama. Let’s talk about creating Screen Studio recordings when you’re back online, Mano.